specificatie/naam.yaml

```diff
@@ -0,0 +1,173 @@
+          $ref: '#/components/schemas/Voornamen'
+        voorvoegsel:
+          $ref: '#/components/schemas/Voorvoegsel'
+        geslachtsnaam:
+          $ref: '#/components/schemas/Geslachtsnaam'
+        adellijkeTitelPredikaat:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+    GbaNaam:
+      allOf:
+        - $ref: '#/components/schemas/GbaNaamBasis'
+        - properties:
+            inOnderzoek:
+              $ref: 'gba-inonderzoek.yaml#/components/schemas/GbaInOnderzoek'
+    GbaNaamGerelateerde:
+      allOf:
+        - $ref: '#/components/schemas/GbaNaamBasis'
+        - type: object
+          properties:
+            inOnderzoek:
+              $ref: 'gba-inonderzoek.yaml#/components/schemas/GbaInOnderzoek'
+    GbaNaamPersoon:
+      allOf:
+        - $ref: '#/components/schemas/GbaNaamBasis'
+        - properties:
+            aanduidingNaamgebruik:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            inOnderzoek:
+              $ref: 'gba-inonderzoek.yaml#/components/schemas/GbaInOnderzoek'
+    NaamBasis:
+      type: object
+      properties:
+        geslachtsnaam:
+          $ref: '#/components/schemas/Geslachtsnaam'
+        volledigeNaam:
+          $ref: '#/components/schemas/VolledigeNaam'
+        voorletters:
+          $ref: '#/components/schemas/Voorletters'
+        voornamen:
+          $ref: '#/components/schemas/Voornamen'
+        voorvoegsel:
+          $ref: '#/components/schemas/Voorvoegsel'
+        adellijkeTitelPredikaat:
+          $ref: '#/components/schemas/WaardetabelMetSoort'
+    Naam:
+      allOf:
+        - $ref: '#/components/schemas/NaamBasis'
+        - properties:
+            inOnderzoek:
+              $ref: '#/components/schemas/NaamInOnderzoek'
+    NaamGerelateerde:
+      type: object
+      properties:
+        voornamen:
+          $ref: '#/components/schemas/Voornamen'
+        voorvoegsel:
+          $ref: '#/components/schemas/Voorvoegsel'
+        geslachtsnaam:
+          $ref: '#/components/schemas/Geslachtsnaam'
+        adellijkeTitelPredikaat:
+          $ref: '#/components/schemas/WaardetabelMetSoort'
+        voorletters:
+          $ref: '#/components/schemas/Voorletters'
+        inOnderzoek:
+          $ref: '#/components/schemas/NaamInOnderzoek'
+    NaamInOnderzoek:
+      description: |
+        Geeft aan welke gegevens over de naam in onderzoek zijn. Zie de [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.1.0/features/in_onderzoek.feature)
+      type: object
+      properties:
+        geslachtsnaam:
+          type: boolean
+        voornamen:
+          type: boolean
+        voorvoegsel:
+          type: boolean
+        adellijkeTitel:
+          type: boolean
+        predikaat:
```

Comment:
we hebben tijdelijk adellijkeTitelPredikaat gesplitst in twee velden, maar dat later teruggedraaid. Dat moet in inOnderzoek nog worden teruggedraaid, want nu zijn het nog twee aparte velden

Reply:
wordt teruggedraaid.